vanilla-framework/vanilla-framework
# New navigation

## Done
Built out the two new navigation styles referencing the spec. Removed the fixed-width and full-width documentation and combined them into a note on the navigation documentation. Created a light and dark navigation example with documentation.

## QA
- Pull code
- Run `gulp test` and check there are no lint errors
- Run `gulp jekyll`
- Go to http://127.0.0.1:4000/vanilla-framework/examples/patterns/navigation/light/ and http://127.0.0.1:4000/vanilla-framework/examples/patterns/navigation/dark/
- Check that they both match the [design specs](https://github.com/ubuntudesign/vanilla-design/tree/master/Navigation)

## Details
Fixes: https://github.com/vanilla-framework/vanilla-framework/issues/1049
